docs/source/alfacase/03_alfacase_example.rst

```diff
@@ -0,0 +1,396 @@
+.. _alfacase-example:
+
+ALFAcase Example
+================
+
+In this section, we will walk trough the creation of a simple network, with two nodes and one pipe, to illustrate how to
+crete a project and how to interpret the definitions available on :ref:`alfacase-reference-section`.
+
+The root level of our :program:`ALFAcase` file is always a :class:`CaseDescription <alfasim_sdk.CaseDescription>`
+which has the following attributes:
+
+.. code-block:: yaml
+
+    name: string    # optional
+    physics: physics_description_schema⠀
+    time_options: time_options_description_schema⠀
+    numerical_options: numerical_options_description_schema⠀
+    ipr_models: ipr_models_description_schema⠀
+    pvt_models: pvt_models_description_schema⠀
+    tracers: tracers_description_schema⠀
+    outputs: case_output_description_schema⠀
+
+    ....
+
+In the following section, this steps will be covered:
+
+- How to configure the project options.
+- How to add a PVtModel.
+- How to add Nodes.
+- How to add Pipes.
+- How to add output curves.
+
+
+.. note::
+
+    For the full reference check the :ref:`alfacase-reference-section`
+
+
+Project Options
+---------------
+
+We will start teh example by configuring the project options, as discussed in the previous section,
+the root level of an ``ALFacase`` file is the :class:`CaseDescription <alfasim_sdk.CaseDescription>` and it has the following options:
```

Comment:
ALFacase with wrong casing, should be
```suggestion
the root level of an :program:`ALFAcase` file is the :class:`CaseDescription <alfasim_sdk.CaseDescription>` and it has the following options:
```
By the way, what does this :program: tag mean?

Reply:
The short answer is font style.

The role `:program:` made more sense on the previous theme, which gave a special highlight for this name. But with this theme, it only applies a bold, which is kind doesn't look good in my opinion =/